docs/Fields.md

```diff
@@ -482,12 +482,10 @@ Here is a custom field displaying the full name:
 ```jsx
 import { useRecordContext } from 'react-admin';
 
-export const FullNameField = (props) => {
+export const FullNameField = ({ label = 'Name', ...props }) => {
```

Comment:
This doesn't work. `Datagrid` and `SimpleShowLayout` inspect the label prop of their child, and a default value defined as you describe cannot be read by the parent.